src/App.tsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import Demo from './MUI_demo';
 
 function App() {
   return (
     <div className="App">
+            <Demo />
```

Comment:
please add component in routing. Example: https://v5.reactrouter.com/web/example/basic